Misc/NEWS.d/next/Library/2017-11-24-08-35-43.bpo-26439.IC45_f.rst

```diff
@@ -0,0 +1,10 @@
+Fix ctypes.util.find_library() for AIX
```

Comment:
I'm not sure of this summary. In short, I understand that you *implemented* find_library() on AIX, not just "fixed".

Reply:
Maybe it just needs to be implemented - I take that to be the "reviews" position.

However, ctypes.util.find_library() does exist for AIX - asis. And, this is broken.
The documentation makes clear that the behavior of find_library is platform-dependent.

So if you object to saying that this patch 'fixes' the published (aka existing feature of ctypes.util) because this patch 'implements' (as ctypes._aix.find_library(), just as "darwin" implements it as ctypes.macholib.dyld.dyld_find() (this was actually what I tried to emulate, just without a new sub-directory, and per much earlier reviews renamed it ctypes._aix to make clear it is not something to be imported automagically. Again - it is a platform dependency (and your suggestion to just import it into the namespace is excellent. Should have thought of that long ago.)

In closing - I see ctypes.util.find_library() as the API definition, and the current implementation is broken. This patch replaces that implementation with a new one - and this 'fixes' the API for AIX.
If the reviews feel that the word 'fixed' is misplaced - I'll delete the line. Or, could/should I say I fixed the AIX implementation of find_library().